packages/shared/src/components/Markdown.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { ReactElement } from 'react';
+import { sanitize } from 'dompurify';
+import styles from './markdown.module.css';
+
+export default function Markdown({
+  children,
+}: {
+  children: string;
```

Comment:
I think that replacing the children's type or using it for the markdown is not a convention. It might be confusing.
Please use a different name.